peps/pep-0768.rst

```diff
@@ -294,6 +294,24 @@ An example usage of the API would look like:
     except Exception as e:
         print(f"Failed to execute code: {e}")
 
+Configuration API
+-----------------
+
+To allow redistributors, system administrators, or users to disable this
+mechanism, several methods will be provided to control the behavior of the
+interpreter:
+
+A new environment ``PYTHON_DISABLE_REMOTE_DEBUGGING`` environment variable will
+be provided to control the behaviour at runtime. If set to any value, the
+interpreter will ignore any attempts to attach a debugger using this mechanism.
+
+This environment variable will be added together with a new ``-X disable-remote-debugging``
+flag to the Python interpreter to allow users to disable this feature at runtime.
+
+Additionally a new  ``--without-remote-debugging`` flag will be added to the
+``configure`` script to allow redistributors to build Python without support for
+remote debugging if they so desire.
```

Comment:
We should additionally expose a flag indicating that remote debugging is disabled via the debug offsets, so that debuggers can check this and print out a useful error message explaining why they won't work, instead of believing that they have attached and then never having their script run. That will provide much nicer end user experience. All 3 methods (env var, `-X`, and `--without-remote-debugging`) should set that flag.